packages/api-request-builder/src/default-params.js

```diff
@@ -68,7 +68,7 @@ export function getDefaultSearchParams(): ServiceBuilderDefaultParams {
  */
 export function setDefaultParams() {
   this.params.expand = getDefaultQueryParams().expand
-  
+
```

Comment:
not sure how this one ended up here 😄

Reply:
I think the git hook doesn't work properly